pyvista/jupyter/__init__.py

```diff
@@ -84,17 +84,26 @@ def set_jupyter_backend(backend):
     except ImportError:  # pragma: no cover
         raise ImportError('Install IPython to display with pyvista in a notebook.')
 
+    if backend == 'ipyvtk_simple':
+        try:
+            import ipyvtklink
+        except ImportError:
+            raise ImportError('Please install `ipyvtklink`. `ipyvtk_simple` '
+                              'is deprecated.')
+        else:
+            backend = 'ipyvtklink'
```

Comment:
@akaszynski, is this well handled in your opinion?

Should we raise a warning here if the user keeps using the `ipyvtk_simple` name?

Reply:
Error is fine.  Usage of `ipyvtk_simple` as a backend was never in our API, so it's not even a depreciation error right now.  We can leave this in, but it's not even necessary at this point.